app/src/main/kotlin/org/fossify/gallery/activities/ViewPagerActivity.kt

```diff
@@ -503,6 +503,7 @@ class ViewPagerActivity : SimpleActivity(), ViewPager.OnPageChangeListener, View
                     }
 
                     hideSystemUI(true)
+                    hideImageInfoPanel()
```

Comment:
We only need the following:

```suggestion
                    if (!mIsFullScreen) {
                        mIsFullScreen = true
                        fullscreenToggled()
                    }
```

This way, we'll still have the animation and no more code duplication.